### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by alexkaserbacher** [TNG/ApiCenter#11]
Implement an architecture that allows to send specific notifications to specific users/user groups. The notification should be triggered in the backend and shown in the frontend - they should be stored per user to show the user a list of all received notifications.

**Comment by fdw** [comment]
I think the mechanism should allow for different paths: One is certainly the frontend, but I can also see emails as another (configurable) one.

**Comment by alexkaserbacher** [comment]
@fdw or more general: allow arbitrary channels to deliver notifications

**Comment by fdw** [comment]
Exactly. Frontend and emails are just the most obvious/important ones.
